src/clj/fluree/db/storage.cljc

```diff
@@ -34,6 +34,10 @@
      :method method
      :local  local}))
 
+(defn parse-local-path
```

Comment:
Does this get used anywhere?

Reply:
It does not get used (yet). I have another branch that implements this function and uses it, and I have some as yet un checked in index exploration code (similar to Brian's excellent [commit exploration code](https://github.com/fluree/db/blob/0f557334b9dd77179c1d91bd36745584ab3e5edb/dev/commit_explore.clj#L1)) that also uses it. I don't think the index exploration code is quite ready for committing yet, and I was going to remove this function, but I decided to leave it in since I know it will soon be useful, but I'm happy to take it out if you or anyone else disagrees.